src/Compilers/Core/MSBuildTaskTests/CscTests.cs

```diff
@@ -113,8 +113,12 @@ public void ChecksumAlgorithmOption()
         public void InstrumentTestNamesFlag()
```

Comment:
Adding a test that specifies multiple kinds exposed a bug in how the option was generated. Fixed in the next commit.